### PR TITLE
[VI] Update README

### PR DESCRIPTION
**Pull request by duythanhvn**
Hi Billy,

I have a question above docs@tensorflow.org list, you can see **[here](https://groups.google.com/a/tensorflow.org/g/docs/c/m1FDWWTBagI/m/XhSabJEIAgAJ)**. It is an opinion of mine, if this is supported, I will update it at this section before merging.

### CODE REVIEW

File: site/vi/README.md [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
-[docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
-
-**Lưu ý:** Chỉ tập trung mọi cơ hội có thể dịch thuật vào nội dung [TensorFlow 2](https://www.tensorflow.org) này trong đường dẫn [site/en](https://github.com/tensorflow/docs/tree/master/site/en). Tài liệu cộng đồng TF 1.x trong [site/en/r1](https://github.com/tensorflow/docs/tree/master/site/en/r1) sẽ không còn được cập nhật khi chúng tôi chuẩn bị ra mắt phiên bản 2.0. Xem thêm thông báo [tại đây](https://groups.google.com/a/tensorflow.org/d/msg/docs/vO0gQnEXcSM/YK_ybv7tBQAJ).
+Những bản dịch này được hoàn thiện dựa trên sự nỗ lực đóng góp từ cộng đồng lập trình viên sử dụng TensorFlow,
```
Comment (lamberta) [inline site/vi/README.md:6]: If changing the disclaimer text, we will need to update it in the notebooks: https://github.com/tensorflow/docs-l10n/tree/master/site/vi/tutorials

Comment (duythanhvn) [reply site/vi/README.md:6]: Sure, I will update this information.

Comment (duythanhvn) [reply site/vi/README.md:6]: @lamberta I would like to ask that this disclaimer text always appears in all documents translated by the Vietnamese community, right?

**Comment by tlimaf** [comment]
@duythanhvn Is there any update on sub-forum issue?

**Comment by duythanhvn** [comment]
@tlimaf truly I don't know yet.

@lamberta Earlier this month, I asked about the sub-forum for the Vietnamese community, which you can review **[here](https://groups.google.com/a/tensorflow.org/g/docs/c/m1FDWWTBagI/m/8npTskudAwAJ)**. I don't know how this is going? Can you update the issue with us?

**Comment by lamberta** [comment]
Yes. Just checked and will be available soon.
In the meantime, can someone please translate the following for the group description ...

```
A group for coordinating the community of amazing contributors to the [TensorFlow documentation](https://www.tensorflow.org/overview).

Questions about usage of TensorFlow are best addressed elsewhere: [Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow), or the [discuss@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/discuss) mailing list.

Send pull requests to [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n).
For guidance, see the [TensorFlow docs contributor guide](https://www.tensorflow.org/community/contribute/documentation), the [Google developer docs style guide](https://developers.google.com/style/highlights), and join the [TensorFlow docs group](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).

And please respect the [TensorFlow code of conduct](https://github.com/tensorflow/tensorflow/blob/master/CODE_OF_CONDUCT.md).
```

**Comment by lamberta** [comment]
Also the title, please: `TensorFlow Vietnamese Documentation Contributors`

**Comment by lamberta** [comment]
Group created: https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-vi
Please join :)

I can add it to the community list on tensorflow.org when the title/description have been translated. Thanks

**Comment by duythanhvn** [comment]
Yes, sure. I can handle this.

> Also the title, please: `TensorFlow Vietnamese Documentation Contributors`

Nhóm Dịch thuật Tài liệu Tensorflow tại Việt Nam

> Yes. Just checked and will be available soon.
> In the meantime, can someone please translate the following for the group description ...
> 
> ```
> A group for coordinating the community of amazing contributors to the [TensorFlow documentation](https://www.tensorflow.org/overview).
> 
> Questions about usage of TensorFlow are best addressed elsewhere: [Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow), or the [discuss@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/discuss) mailing list.
> 
> Send pull requests to [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n).
> For guidance, see the [TensorFlow docs contributor guide](https://www.tensorflow.org/community/contribute/documentation), the [Google developer docs style guide](https://developers.google.com/style/highlights), and join the [TensorFlow docs group](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
> 
> And please respect the [TensorFlow code of conduct](https://github.com/tensorflow/tensorflow/blob/master/CODE_OF_CONDUCT.md).
> ```

Đây là nhóm điều phối Cộng đồng và thảo luận dành cho những người dịch thuật tuyệt vời của [Tài liệu TensorFlow](https://www.tensorflow.org/overview).

Những câu hỏi về việc sử dụng TensorFlow được hỗ trợ tốt nhất tại: [Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow), hoặc tại Diễn đàn [discuss@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/discuss).

Hãy gửi Pull request đến [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n).

Để hiểu rõ hơn, hãy xem qua [Tài liệu Hướng dẫn Cộng tác viên của TensorFlow](https://www.tensorflow.org/community/contribute/documentation), [Tài liệu Style Guide dành cho Nhà phát triển của Google](https://developers.google.com/style/highlights), và tham gia [Nhóm Tài liệu TensorFlow](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).

Cuối cùng, vui lòng tôn trọng [Những quy tắc Ứng xử trong Cộng đồng TensorFlow](https://github.com/tensorflow/tensorflow/blob/master/CODE_OF_CONDUCT.md).

**Comment by lamberta** [comment]
Thank you
Should be good now. I'll send a follow-up message on the group thread

**Comment by duythanhvn** [comment]
Did you mean to restore the REVIEWERS file back to its original state? The members listed in this list are no longer active, I deleted their names to avoid disturbing them on Github.

**Comment by lamberta** [comment]
Ok. In this pull request, let's only add your name. Then make a separate pull request to remove inactive reviewers---and double-check it's fine with them. There's no real harm in pinging extra reviewers if they're ok with it. But, yes, it does make it clearer to me who to add if an issue comes up (like a broken notebook, etc.)

cc: @truongsinh, @truongnh1992, @anniehiehie

**Comment by duythanhvn** [comment]
Sure, let's do that. I will update another PR for this.

**Comment by tlimaf** [comment]
@duythanhvn Please add my name in the next update. Thanks!

**Comment by lamberta** [comment]
@tlimaf You can add yourself—actually, it's preferred :)
That way we can verify you've signed the CLA, etc.

File: site/vi/tutorials/keras/overfit_and_underfit.ipynb [hunk 2]
```diff
@@ -133,6 +148,13 @@
       },
       "outputs": [],
       "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "\n",
+        "try:\n",
+        "  # %tensorflow_version only exists in Colab.\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n",
```
Comment (lamberta) [inline site/vi/tutorials/keras/overfit_and_underfit.ipynb:157]: this section no longer in source notebook

**Comment by lamberta** [comment]
Thanks. I rebased to fix the merge conflict and ran the notebook through [nbfmt.py](https://github.com/tensorflow/docs/blob/master/tools/nbfmt.py) which is required to pass the CI tests.

**Pull request merged**